account_bank_statement_line_reconciliation/wizard/account_bank_statement_line_reconciliation_wizard.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class AccountBankStatementLineReconciliationWizard(models.TransientModel):
+    _name = "account.bank.statement.line.reconciliation.wizard"
+
+    def _account_move_ids_and_journal_id(self):
+        ids = self._context.get('active_ids')
+        if ids:
+            account_move_ids = self.env['account.move'].browse(ids)
+
+            journal_ids = list(set(account_move_ids.mapped('journal_id.id')))
+
+            if len(account_move_ids) > 1 and len(journal_ids) > 1:
```

Comment:
Are you sure of your condition? We want a `and` condition or a `or` condition?

Reply:
We need `and`. Because a user can select several `account.move`s and this moves can have different `journal`s. So condition is if we have several moves `and` several journals then warn the user.